src/geometry/manifold/diffeomorph.lean

```diff
@@ -0,0 +1,81 @@
+/-
+Copyright © 2020 Nicolò Cavalleri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Nicolò Cavalleri.
+-/
+
+import geometry.manifold.times_cont_mdiff_map
+
+/-!
+# Diffeomorphisms
+This file implements diffeomorphisms.
+
+-/
+
+open_locale manifold
+
+variables {𝕜 : Type*} [nondiscrete_normed_field 𝕜]
+{E : Type*} [normed_group E] [normed_space 𝕜 E]
+{E' : Type*} [normed_group E'] [normed_space 𝕜 E']
+{F : Type*} [normed_group F] [normed_space 𝕜 F]
+{H : Type*} [topological_space H]
+{H' : Type*} [topological_space H']
+{G : Type*} [topological_space G]
+(I : model_with_corners 𝕜 E H) (I' : model_with_corners 𝕜 E' H')
+(J : model_with_corners 𝕜 F G)
+
+section diffeomorph
+
+variables (M : Type*) [topological_space M] [charted_space H M] [smooth_manifold_with_corners I M]
+(M' : Type*) [topological_space M'] [charted_space H' M'] [smooth_manifold_with_corners I' M']
+(N : Type*) [topological_space N] [charted_space G N] [smooth_manifold_with_corners J N]
+(n : with_top ℕ)
+
+/-- M and M' are diffeomorph with respect to I and I' -/
```

Comment:
The docstring seems to imply that this definition is Prop, i.e., the existence of a diffeomorphism, but in fact the definition is data. Can you change the docstring? Also, write `M` instead of M, and so on.

Reply:
Sure, even though it is this way because most of the file is copied from topology modulo changing homeomorph with diffeomorph, so we should change it there as well I guess